tests/python/gaia-ui-tests/gaiatest/apps/search/app.py

```diff
@@ -33,15 +40,9 @@ def go_to_url(self, url):
 
     @property
     def history_items_count(self):
+        Wait(self.marionette).until(expected.element_present(*self._history_locator))
         return len(self.marionette.find_elements(*self._history_item_locator))
 
     def wait_for_history_to_load(self, number_of_items=1):
+        Wait(self.marionette).until(expected.element_present(*self._history_locator))
```

Comment:
Yeah, I guess I could do that, although it would do a lot of overly checking of _history_locator element present, but I guess that's not really a problem.